js/cex.js

```diff
@@ -859,6 +859,9 @@ module.exports = class cex extends Exchange {
     }
 
     async fetchOrder (id, symbol = undefined, params = {}) {
+        if (id === undefined) {
```

Comment:
We don't think this check is necessary, since `id` is a required argument for all `fetchOrder` implementations. In other words, this check is redundant. Do you agree?